qa/suites/rgw/lifecycle/tasks/rgw_s3tests.yaml

```diff
@@ -7,14 +7,5 @@ tasks:
     client.0:
       rgw_server: client.0
       storage classes: LUKEWARM, FROZEN
-      extra_attrs: ["lifecycle"]
```

Comment:
this change (running all s3tests instead of just `lifecycle` ones) uncovered some failures related to storage classes:
```
=========================== short test summary info ============================
FAILED s3tests/functional/test_s3.py::test_object_storage_class - assert Fals...
FAILED s3tests/functional/test_s3.py::test_object_storage_class_multipart - A...
FAILED s3tests/functional/test_s3.py::test_object_modify_storage_class - asse...
FAILED s3tests/functional/test_s3.py::test_object_modify_storage_class_multipart
FAILED s3tests/functional/test_s3.py::test_object_storage_class_copy - assert...
FAILED s3tests/functional/test_s3.py::test_object_storage_class_copy_multipart
```
it looks like rgw/lifecycle suite was the only subsuite that actually configured s3tests with storage classes